source/site-administrators/engine/engine-headers-based-authentication.rst

```diff
@@ -169,8 +169,8 @@ To add optional attributes, add the following inside the ``<headers>`` tag:
           <!-- Optional attribute mappings, allows to expose attributes from the external auth -->
           <attributes>
             <attribute>
-              <name>APP_ATTRIBUTE_NAME</name> <!-- The name of the attribute in the header -->
-              <field>name</field>             <!-- The name of the attribute in the authentication object -->
+              <name>APP_ATTRIBUTE_NAME</name>   <!-- The name of the attribute in the header, excluding the prefix -->
+              <field>YOUR_FIELD_NAME</field>    <!-- The name of the attribute in the authentication object -->
```

Comment:
We try to make code snippets ready for copy/paste so people can try it. I think it's better to keep the actual name used in the example instead of a placeholder.

Reply:
Ok. That's what we have now, right?

Reply:
Other places of this guide already use `name` for example
```ftl
<#if (authToken.principal)??>
      <#assign name = authToken.principal.attributes.name!"stranger" />
<#else>
```

Reply:
Maybe that's the confusion, `name` was not supposed to be a placeholder but the actual value people would use to try it out following the guide

Reply:
The guide was build using the existing code from our editorial blueprint (again so it's easier to copy/pase and see it working) so that would mean changing the blueprint too

Reply:
Yes, thanks